understand/observed-size-spectra.qmd

```diff
@@ -432,7 +423,7 @@ Now make a size-spectrum plot for a new data set. If you happen to have access t
 
 4.  To avoid the dependence on bin sizes, we use densities, where the total number of individuals or the total biomass of individuals in each bin are divided by the width of the bin. We refer to these as the number density or the biomass density respectively.
 
-5.  When we work with a logarithmic weight axis, then it is natural to use densities in log weight, where the numbers of individuals in each bin are divided by the log (**ASTA - what log? log 10, natural log or log 2?)** of the width of the bin. We refer to these as the number density in log weight or the biomass density in log weight respectively.
```

Comment:
Changing the base of the logarithm only scales the density by a constant. As we are currently more interested in the shape of the size spectra than their numerical values, I don't think we need to go into this detail.